docs/rules/no-children-prop.md

```diff
@@ -0,0 +1,21 @@
+# Prevent passing of children as props (no-children-prop)
+
+Children should always be actual children, not passed in as a prop.
+
+## Rule Details
+
+The following patterns are considered warnings:
+
+```js
```

Comment:
Oh? I usually see lots of blocks with red backgrounds when using `js` on JSX code. Here's an example.

Before: https://github.com/Khan/aphrodite/tree/436325f8848db974e6f9ffba18afa6af1c426095#api
After: https://github.com/Khan/aphrodite/tree/a1fdb520fd9fcc50148a30de2c49c7c1253a604e#api